README.md

```diff
@@ -40,6 +40,7 @@ ___
 - [File](#File)
   - [Get a File](#Get-a-File)
   - [Get a File as base64](#Get-a-File-as-base64)
+  - [Upload a File as base64](#Upload-a-File-as-base64)
```

Comment:
Did you use `npm run doc`? It should have generated more than this.

Reply:
Good shout, didn't know about that command. Running it right now seems to tag my current branch name, but it did show me what params I needed to document better